tap_dixa/exceptions.py

```diff
@@ -20,15 +20,15 @@ def __init__(self, message=None, response=None):
 class DixaClient5xxError(DixaClientError):
     pass
```

Comment:
Move DixaClient5xxError class to below DixaClient4xxError.

Reply:
Done